### PR TITLE
Readme: Provide instructions if used custom model naming for Azure OpenAI

### PR DESCRIPTION
**Pull request by momuno**
### Motivation and Context

Please help reviewers and future users, providing the following information:
  1. Why is this change required? alerts users to problem encountered and what to do about it.
  2. What problem does it solve?  chat copilot returned error during chat due to Azure OpenAI custom deployed model names not being used by configure script, 
  3. What scenario does it contribute to? running the quickstart guide scripts for Azure OpenAI when a user has deployed a model with a custom name.
  4. If it fixes an open issue, please link to the issue here.

### Description

Added a note about when using custom deployed names for Azure OpenAI, what parameters to pass to the configure scripts.

### Contribution Checklist

- [x] The code builds clean without any errors or warnings
- [x] The PR follows the [Contribution Guidelines](https://github.com/microsoft/copilot-chat/blob/main/CONTRIBUTING.md) and the [pre-submission formatting script](https://github.com/microsoft/copilot-chat/blob/main/CONTRIBUTING.md#development-scripts) raises no violations
- [x] All unit tests pass, and I have added new tests where possible
- [x] I didn't break anyone :smile:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,15 +53,21 @@ You will need the following items to run the sample:
...
+        ```
+
 4. Run Chat Copilot locally. This step starts both the backend API and frontend application.
     
     ```powershell
     .\Start.ps1 
     ```
 
-    > **IMPORTANT:** Confirm pop-ups are not bocked and you are logged in with the same account used to register the application.
+    > NOTE: Confirm pop-ups are not bocked and you are logged in with the same account used to register the application.
```
Comment (glahaye) [inline README.md:68]:
```suggestion
    > NOTE: Confirm pop-ups are not blocked and you are logged in with the same account used to register the application.
```

File: README.md [hunk 2]
```diff
@@ -87,15 +93,21 @@ You will need the following items to run the sample:
...
+        ```
+
 4. Run Chat Copilot locally. This step starts both the backend API and frontend application.
 
     ```bash
     ./Start.sh
     ```
 
-    > **IMPORTANT:** Confirm pop-ups are not bocked and you are logged in with the same account used to register the application.
+    > NOTE: Confirm pop-ups are not bocked and you are logged in with the same account used to register the application.
```
Comment (glahaye) [inline README.md:108]:
```suggestion
    > NOTE: Confirm pop-ups are not blocked and you are logged in with the same account used to register the application.
```

File: README.md [hunk 3]
```diff
@@ -121,15 +133,21 @@ You will need the following items to run the sample:
...
+        ```
+
+4. Run Chat Copilot locally. This step starts both the backend API and frontend application.
 
     ```bash
     ./Start.sh
     ```
 
-    > **IMPORTANT:** Confirm pop-ups are not bocked and you are logged in with the same account used to register the application.
+    > NOTE: Confirm pop-ups are not bocked and you are logged in with the same account used to register the application.
```
Comment (glahaye) [inline README.md:148]:
```suggestion
    > NOTE: Confirm pop-ups are not blocked and you are logged in with the same account used to register the application.
```

**Review by glahaye — DISMISSED** [review]

**Review by crickman — APPROVED** [review]

**Comment by matthewbolanos** [comment]
Before this can be merged, there also needs to be an analogous PR for docs so that the instructions there are update.
- Docs page: https://learn.microsoft.com/en-us/semantic-kernel/chat-copilot/getting-started

File: scripts/Configure.sh [hunk 4]
```diff
@@ -32,17 +32,17 @@ while [[ $# -gt 0 ]]; do
       shift
       shift
       ;;
-    --completion)
+    --completionmodel)
```
Comment (matthewbolanos) [inline scripts/Configure.sh:35]: This impacts the docs on learn. Please submit a PR to fix that before this is merged.

Comment (momuno) [reply scripts/Configure.sh:35]: oh snap.  where do I submit a PR for learn docs?

Comment (glahaye) [reply scripts/Configure.sh:35]: @matthewbolanos If you are talking about what's in https://github.com/MicrosoftDocs/semantic-kernel-pr/ I couldn't find references to the "completion" parameter in those documents and we should be good to go.

Are there other documents??

**Review by glahaye — APPROVED** [review]

File: scripts/Configure.sh (hunk 4 above)
Comment (matthewbolanos) [reply scripts/Configure.sh:35]: This PR https://github.com/MicrosoftDocs/semantic-kernel-pr/pull/134 looks great :) Thanks @momuno. Let me know when this PR is merged in and then I'll do the same on the docs.

**Review by alliscode — APPROVED** [review]

**Pull request merged**